GitTfs/Core/IGitTfsRemote.cs

```diff
@@ -19,6 +19,13 @@ public interface IRenameResult
         string LastParentCommitBeforeRename { get; set; }
     }
 
+    // Internal WorItem representation for export scenarios
+    public interface IExportWorkItem
+    {
+        string Id { get; set; }
+        string Title { get; set; }
```

Comment:
The `set` methods can be removed. I don't think there's anything that uses the interface that needs to (or should) update the work item.